pexpect/expect.py

```diff
@@ -157,8 +168,8 @@ def __str__(self):
         ss = list(zip(*ss))[1]
         return '\n'.join(ss)
 
-    def search(self, buffer, freshlen, searchwindowsize=None):
-        '''This searches 'buffer' for the first occurrence of one of the search
+    def search(self, buff, freshlen, searchwindowsize=None):
```

Comment:
@takluyver related to your comment, I just reverted these - the changes I made ended up being unnecessary.

Reply:
Great, thanks. Could you also revert `buff` back to `buffer` in the parameters? Any parameter in Python can be passed as a keyword argument, so people might be relying on the name.

Reply:
Good point, I'll fix that.